.gitignore

```diff
@@ -1,2 +1,5 @@
 target
 work
+.project
```

Comment:
I will leave it to users of IDEs which require proprietary files to be added to Maven repositories to waste time added those ignores. :-)